### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by Smolevich**
- Update travis ci matrix(add PHP 7.3) (removed from this PR, will be moved in another PR)
- Update Dockerfile, use alpine image instead debian image (removed from this PR, will be moved in another PR)
- Fix tests
- Update composer.json

### CODE REVIEW

**Comment by Smolevich** [comment]
@jenssegers , can you review please?

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+RUN apk add --update --no-cache -t .php-build-deps \
+    autoconf \
+    libmcrypt=2.5.8-r7 \
+    git \
+    make \
+    g++ \
+    gcc \
+    openssl-dev \
+    libzip-dev; \
+    pecl install mongodb-1.5.3
```
Comment (jenssegers) [inline Dockerfile]: Do we need this fixed version?

Comment (jenssegers) [reply Dockerfile]: Also can't we still use the docker-php-ext-install?

**Comment by Smolevich** [comment]
> Any chance we can separate the Travis changes and the fixes? Or are the Travis changed required?



> Any chance we can separate the Travis changes and the fixes? Or are the Travis changed required?

It is possible, but better change all in this PR

File: Dockerfile (hunk 1 above)
Comment (Smolevich) [reply Dockerfile]: mongodb-1.5.3 latest stable version in https://pecl.php.net/package/mongodb

Comment (Smolevich) [reply Dockerfile]: In original Dockerfile used https://github.com/jenssegers/laravel-mongodb/blob/master/Dockerfile#L10
`pecl install mongodb && docker-php-ext-enable mongodb`

**Comment by Smolevich** [comment]
@jenssegers, i can remove fix tests from this PR and move in another PR for understanding. Can you review this changes tomorrow?

**Comment by jenssegers** [comment]
Yes, would rather have these 2 things as separate PR's if possible 👌

**Comment by Smolevich** [comment]
@jenssegers, for this PR i delete changes for travis ci

**Comment by Smolevich** [comment]
Changes for Dockerfile will be changed in another PR?

**Comment by Smolevich** [comment]
@jenssegers, for this PR i delete changes for travis ci and docker-compose and move in another PR. Please see both PR tomorrow

File: docker-compose.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ services:
         volumes:
             - .:/code
         working_dir: /code
-        command: bash -c "composer install --prefer-source --no-interaction && php ./vendor/bin/phpunit"
```
Comment (jenssegers) [inline docker-compose.yml]: Can we also revert these changes please?

Comment (Smolevich) [reply docker-compose.yml]: @jenssegers, I reverted this changes
